app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -347,9 +347,9 @@ class ExplorationActivityTest {
     explorationDataController.stopPlayingExploration()
   }
 
-  // TODO (#1855): Resolve ktlint max line in app module test
+  
   @Test
-  fun testAudioWithCellular_openRatioExploration_clickAudioIcon_changeConfiguration_checkOpensCellularAudioDialog() { // ktlint-disable max-line-length
+  fun testAudWithCell_openRatioExp_clickAudIcon_changeConfig_checkOpensCellAudDialog() {
```

Comment:
this is doesn't sounds good `testAudWithCell`

`testAudioWithCellular_ratioExp_audioIcon_configChange_opensCellAudioDialog`
Let me know if this works, and also keep consistency with `configChange` everywhere.

Reply:
follow a similar pattern with others as well.